### PR TITLE
update Freifunk rules

### PR DESCRIPTION
**Pull request by anoymouserver**
add and update communities

### CODE REVIEW

File: src/chrome/content/rules/Freifunk.xml [hunk 1]
```diff
@@ -67,6 +71,20 @@
 		to="https://www.youtube.com/user/FREIFUNK2013" />
 	<test url="http://youtube.freifunk.net/?v=RoKQQ5a6E1Q" />
 
+	<!-- Redirect communities to their own domains: -->
+	<rule from="^http://(?:gera|greiz|gera-greiz)\.freifunk\.net/(.*)"
```
Comment (J0WI) [inline src/chrome/content/rules/Freifunk.xml]: Please avoid non-capturing groups and add a note that this redirect is required because of a mismatch.

Comment (J0WI) [inline src/chrome/content/rules/Freifunk.xml]: Please also remove the `(.*)` and `$1`, this is done automatically by HTTPS Everywhere.

**Comment by anoymouserver** [comment]
I've informed the web team of Freifunk.net because of the missing certificate for `www.freifunk.net`.
Let's see what they will do..

**Comment by anoymouserver** [comment]
The web team doesn't respond, I've removed the `www.` subdomain so it may be re-added if working.

**Comment by J0WI** [comment]
Could you merge the chages from #9991 into this?

File: src/chrome/content/rules/Freifunk.xml [hunk 2]
```diff
@@ -67,6 +71,20 @@
...
+	<!-- Redirect communities to their own domains (because of mismatch): -->
+	<rule from="^http://(gera|greiz|gera-greiz)\.freifunk\.net/"
+		to="https://freifunk-gera-greiz.de/" />
+	<test url="http://gera.freifunk.net/favicon.ico" />
+
+	<rule from="^http://goettingen\.freifunk\.net/"
+		to="https://freifunk-goettingen.de/" />
+	<test url="http://goettingen.freifunk.net/favicon.ico" />
+
+	<rule from="^http://guetersloh\.freifunk\.net/"
```
Comment (J0WI) [inline src/chrome/content/rules/Freifunk.xml]: No mismatch for me here.

File: src/chrome/content/rules/Freifunk.xml [hunk 3]
```diff
@@ -67,6 +71,20 @@
 		to="https://www.youtube.com/user/FREIFUNK2013" />
 	<test url="http://youtube.freifunk.net/?v=RoKQQ5a6E1Q" />
 
+	<!-- Redirect communities to their own domains (because of mismatch): -->
+	<rule from="^http://(gera|greiz|gera-greiz)\.freifunk\.net/"
+		to="https://freifunk-gera-greiz.de/" />
+	<test url="http://gera.freifunk.net/favicon.ico" />
+
+	<rule from="^http://goettingen\.freifunk\.net/"
```
Comment (J0WI) [inline src/chrome/content/rules/Freifunk.xml]: No mismatch for me here.

File: src/chrome/content/rules/Freifunk.xml [hunk 4]
```diff
@@ -26,19 +27,23 @@
 <ruleset name="Freifunk.net">
 
 	<target host="freifunk.net"/>
-	<target host="www.freifunk.net"/>
 	<target host="bug.freifunk.net"/>
 	<target host="facebook.freifunk.net"/>
 	<target host="forum.freifunk.net"/>
-	<target host="blog.guetersloh.freifunk.net"/>
```
Comment (J0WI) [inline src/chrome/content/rules/Freifunk.xml]: Why did you removed this host? It WFM.

**Comment by J0WI** [comment]
And #9982 is also related.

**Comment by J0WI** [comment]
@anoymouserver sorry I didn't notice your changes. Could you please rebase your branch against current master and ping me then?

**Comment by jeremyn** [comment]
@anoymouserver Are you still interested in working on this pull request?

**Comment by anoymouserver** [comment]
I think most (probably all) of the changes have been made with other PRs in the meantime.

**Comment by jeremyn** [comment]
@anoymouserver Thanks for the update.

**Pull request closed without merging**